tools/buildsys/src/builder.rs

```diff
@@ -387,7 +397,18 @@ impl DockerBuild {
             ),
             target_build_args: TargetBuildArgs::Kit(KitBuildArgs {
                 kit: kit.to_string(),
+                vendor: manifest.info().kit_vendor().context(error::GraphSnafu)?,
+                local_kits: serde_json::to_string(
+                    &manifest.kit_dependencies().context(error::GraphSnafu)?,
+                )
```

Comment:
Why are you passing this vec as JSON? Couldn't you just use the list like in the other cases?

```suggestion
                local_kits: manifest.kit_dependencies().context(error::GraphSnafu)?
```